### PR TITLE
Namespace monitoring corrected with examples...

### PR DESCRIPTION
**Pull request by aphorise**
Further to customer feedback

### CODE REVIEW

File: website/pages/docs/internals/limits.mdx [hunk 1]
```diff
@@ -58,9 +58,11 @@ stored separately, so the limit applies to each independently.
 Specifying distinct per-mount options, or using long mount point paths, can
 increase the space required per mount.
 
-The number of mount points can be monitored by reading the
-[`sys/auth`](/api-docs/system/audit) and
-[`sys/mounts`](/api-docs/system/mounts) endpoints, which contain all mount points in all namespaces.
+The number of mount points can be monitored by reading the 
+[`sys/auth`](/api-docs/system/audit) and [`sys/mounts`](/api-docs/system/mounts)
```
Comment (ncabatoff) [inline website/pages/docs/internals/limits.mdx:62]: Huh, just noticed a pre-existing error in this bit: the link should be to api-docs/system/auth, not /audit.

Comment (mgritter) [inline website/pages/docs/internals/limits.mdx:62]:
```suggestion
[`sys/auth`](/api-docs/system/auth) and [`sys/mounts`](/api-docs/system/mounts) 
```

**Review by mgritter — APPROVED** [review]
Thanks for the fix, my mistake-- I confused sys/raw/sys/auth with the sys/auth API, which does filter per namespace.

We do have metrics for these, though, in the next release, and I believe @HridoyRoy made some changes to add them to this section?

**Comment by aphorise** [comment]
This PR is re-done on: #10863.

**Pull request closed without merging**